web/lib/enums/httpEnum.ts

```diff
@@ -0,0 +1,43 @@
+/*
+MIT License
+
+Copyright (c) 2020-present, Vben
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+*/
+
+export enum ContentTypeEnum {
```

Comment:
Is this file copied from other projects?

Reply:
It is better to add a line like "Referred from core/src/main/java/org/apache/iceberg/rest/RESTClient.java" on the top of the code.

All the copied code need this line.

<img width="717" alt="image" src="https://github.com/datastrato/gravitino/assets/850797/a3a7161b-de2e-4987-a052-dcf5319c6809">

Reply:
You also need to update LICENSE and LICENSE.bin

Reply:
Also if it was copied from Iceberg what does the file have an MIT license header?

Reply:
The pasted image is just an example.